types.go

```diff
@@ -138,6 +138,13 @@ type KeyboardButton struct {
 	InlineQuery string `json:"switch_inline_query,omitempty"`
 }
 
+// InlineKeyboardMarkup represents an inline keyboard that appears right next
+// to the message it belongs to.
+type InlineKeyboardMarkup struct {
+	// Array of button rows, each represented by an Array of KeyboardButton objects.
+	InlineKeyboard [][]KeyboardButton `json:"inline_keyboard,omitempty"`
```

Comment:
The `KeyboardButton` type in fact implements [InlineKeyboardButton](https://core.telegram.org/bots/api#inlinekeyboardbutton).

[KeyboardButton](https://core.telegram.org/bots/api#keyboardbutton) is a different entity and not currently represented by anything in telebot. I'd love to correct this but unfortunately that would break the current API.